src/Core/Request.php

```diff
@@ -2,77 +2,15 @@
 
 namespace TrendingRepos\Core;
 
-class Request {
-
-    /**
-     * @var array
-     */
-    public $parameters = [];
-
-
-    /**
-     * Request parameters
-     * @var array
-     */
-    public $params = [];
-
-
-    /**
-     * Request clearn uri
-     * @return mixed|string
-     */
-    public static function uri() {
-        $uri = trim($_SERVER['REQUEST_URI'], '/');
-
-        if(!empty($_SERVER['REQUEST_QUERY'])){
-            $uri= str_replace( $_SERVER['REQUEST_QUERY'], '', $uri );
-        }
-
-        if(!empty($_SERVER['QUERY_STRING'])) {
-            $uri= str_replace( $_SERVER['QUERY_STRING'], '', $uri );
-        }
-
-        $uri = trim($uri, '?');
-
-
-        return $uri;
-
-    }
-
-
-    /**
-     * Return request method type
-     * @return mixed
-     */
-    public static function methodType() {
+class Request 
+{
+    public function methodType(): string 
```

Comment:
As I mentioned earlier, normally method names have to include a verb. Here there's none 🤔